antismash/detection/hmm_detection/cluster_rules.txt

```diff
@@ -278,3 +278,12 @@ RULE RaS-RiPP
     CUTOFF 5
     EXTENT 10
     CONDITIONS PF04055 and TIGR01716
+
+RULE fungal-RiPP
```

Comment:
While we now have a case sensitivity check, I see no added value in making this rule have mixed case.